pkg/nfd-worker/nfd-worker.go

```diff
@@ -140,15 +139,24 @@ func NewNfdWorker(args Args) (NfdWorker, error) {
 		&local.Source{},
 	}
 
-	sourceWhiteList := map[string]struct{}{}
-	for _, s := range args.Sources {
-		sourceWhiteList[strings.TrimSpace(s)] = struct{}{}
-	}
+	// Determine enabled feature
+	if len(args.Sources) == 1 && args.Sources[0] == "all" {
+		nfd.sources = allSources
```

Comment:
We should also remove the `panicfake` source too from `allSources`

Reply:
VERY good note, thanks! Fixed